components/custom-checkbox/web-component/story-template.jsx

```diff
@@ -50,13 +50,17 @@ export const defaultArgs = {
   value: '',
 };
 
+export const exampleArgs = {
+  value: 'I agree',
+};
+
 export const CustomCheckbox = ({
-  checked = false,
-  disabled = false,
-  indeterminate = false,
-  invalid = false,
-  required = false,
-  value = '',
+  checked = defaultArgs.checked,
+  disabled = defaultArgs.disabled,
+  indeterminate = defaultArgs.indeterminate,
+  invalid = defaultArgs.invalid,
+  required = defaultArgs.required,
+  value = defaultArgs.value,
 }) => (
   <utrecht-custom-checkbox
     checked={checked}
```

Comment:
Many `false` attributes show up by default in the code snippet, better fall back to `undefined`:

```
checked="false" disabled="false" indeterminate="false" invalid="false" required="false"
```